menu/navigation.json

```diff
@@ -56,6 +56,10 @@
                     "label": "Recover a lost password",
                     "slug": "recover-a-lost-password"
                   },
+                  {
```

Comment:
you need to add the previous and next buttons at the end of this and all related how-to pages.